pkgs/top-level/config.nix

```
@@ -50,6 +50,18 @@ let
       '';
     };
 
+    showDerivationWarnings = mkOption {
+      type = types.bool;
+      default = false;
```

```suggestion
      type = types.enum [ "hasNoMaintainers" ];
      default = [ ];
```

maybe as a enum so users could pick which warnings they want shown